Sources/DateScrubber.swift

```diff
@@ -9,11 +9,11 @@
 import UIKit
 
 public protocol DateScrubberDelegate {
-     func requestToSetContentView(dateScrubber:DateScrubber, toYPostion yPosition: CGFloat)
+     func requestToSetContentView(dateScrubber:DateScrubber, toYPosition yPosition: CGFloat)
 }
 
 public extension DateScrubberDelegate where Self: UICollectionViewController {
-    func requestToSetContentView(dateScrubber:DateScrubber, toYPostion yPosition: CGFloat){
+    func requestToSetContentView(dateScrubber:DateScrubber, toYPosition yPosition: CGFloat){
 
         self.collectionView?.setContentOffset(CGPointMake(0,yPosition), animated: true)
```

Comment:
Using `CGPoint(x: 0, y: yPosition)` might be better to be consistent with the guidelines.

https://github.com/bakkenbaeck/iOS-playbook/blob/master/SWIFT_STYLE_GUIDE.md#struct-initializers